src/kernel-install/90-loaderentry.install.in

```diff
@@ -115,6 +115,43 @@ install -m 0644 "$KERNEL_IMAGE" "$KERNEL_DEST" || {
 }
 chown root:root "$KERNEL_DEST" || :
 
+if [ -n "$KERNEL_INSTALL_CONF_ROOT" ]; then
+    if [ -f "$KERNEL_INSTALL_CONF_ROOT/devicetree" ]; then
+        read -r DEVICETREE <"$KERNEL_INSTALL_CONF_ROOT/devicetree"
+    fi
+elif [ -f /etc/kernel/devicetree ]; then
+    read -r DEVICETREE </etc/kernel/devicetree
+elif [ -f /usr/lib/kernel/devicetree ]; then
+    read -r DEVICETREE </usr/lib/kernel/devicetree
+fi
+if [ -n "$DEVICETREE" ]; then
+    for prefix in \
+        "/boot/dtb-$KERNEL_VERSION" \
+        "/boot/dtbs/$KERNEL_VERSION" \
```

Comment:
I'm not convinced that reading stuff from `/boot` is useful. We're taking a file from `/boot` and (after some steps) putting it back in `/boot` again. I know some distros follow this scheme, but it's backwards ;)  OTOH, it doesn't hurt, if it's not used.

Reply:
Hmm, thinking about it some more, `/boot` is not trusted, which means that we it actually *is* a problem if a file is loaded from there. So I think we should make this opt-in somehow.

@poettering, @bluca, opinions on this?

Reply:
I don't disagree, but `/boot/dtbs/$KERNEL_VERSION` is what you'll get with just `make install` in the kernel tree :/

Reply:
I'm pretty sure they're knobs in the kernel makefile to override where it puts the dtbs, it provides those knobs for all the other stuff as well. Just because the default is bad doesn't mean we need to accomodate it.